ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,156 @@
+*** Settings ***
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource
+Resource        ../../Resources/ODS.robot
+Suite Setup     Setup Settings Accelerator Profiles
+Suite Teardown  Teardown Settings Accelerator Profiles
+
+
+*** Variables ***
+${ACC_DISPLAY_NAME}=    qe_create_ap_
+${ACC_NAME}=    qecreateap
+${ACC_IDENTIFIER}=    nvidia.com/gpu
+${ACC_DESCRIPTION}=    description example
+${ACC_ENABLED}=    True
+${ACC_TOLERATION_OPERATOR}=    Exists
+${ACC_TOLERATION_EFFECT}=    PreferNoSchedule
+${ACC_TOLERATION_KEY}=    my_key
+${ACC_TOLERATION_VALUE}=    my_value
+${ACC_TOLERATION_SECONDS}=    15
+
+
+*** Test Cases ***
+Create An Accelerator Profile From "Accelerator Profiles" Administration UI
+    [Documentation]    Create an Accelerator Profile instance from the Administration UI and verify it's content
+    [Tags]  RHOAIENG-3349
+    ...     Smoke
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Click On Create Accelerator Profile Button
+    Create An Accelerator Profile Via UI   ${ACC_DISPLAY_NAME}01   ${ACC_IDENTIFIER}
+    ...                                    ${ACC_DESCRIPTION}   ${ACC_ENABLED}    tolerations=yes
+    ...                                    tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                                    tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                                    tol_key=${ACC_TOLERATION_KEY}
+    ...                                    tol_value=${ACC_TOLERATION_VALUE}
+    ...                                    tol_seconds=${ACC_TOLERATION_SECONDS}
+    In The Accelerator Profiles Grid There Is An Accelerator Profile With Name   ${ACC_DISPLAY_NAME}01
```

Comment:
`Accelerator Profile Should Exist` or `Accelerator Profile Should Be Displayed` would be more clear and more consistent with RF style (e.g., Should Be Equal) IMO